CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Added VIP support. VIPs will now show in the !debug command and on the userpage on the website. (#886)
```

Comment:
Also should add that founders support was added

Reply:
Additionally, this data is also present on the API, which should be noted

Reply:
👍